stack-scholar/recipes/deploy.rb

```diff
@@ -1,6 +1,23 @@
 include_recipe 'php-fpm::service'
 include_recipe 'nginx-app::service'
 
+#
+# An attempt to make sure that the php prefix given is the prefix
+# chosen for php-cli at the end of an install
+php_package_prefix = node['php']['ppa']['package_prefix'].gsub(/php/, '')
```

Comment:
@gilleyj I think this should go into the default `php-fpm` cookbook so it's in all stacks. I am assuming that EasyBib, etc. would have the same issue otherwise.